### PR TITLE
Resolve #3154: Random Title Sequence

### LINKED ISSUE #3154: Random Title Sequence
It'd be cool to randomly select title sequence just like how you can randomly select music for it.

### PR DESCRIPTION
**Pull request by MichaelJBerk**
Due to me being an idiot, I had to recreate the develop branch. I'm making a new pull request.

Implements #3154.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Really wish you didn't recreate the pr. We lose the history every time you do that. You can always keep the old pr even with completely different code.

**Comment by MichaelJBerk** [comment]
In the mishap with the commits, the fix to the merge conflict to en-GB.txt didn't go though properly, so I have fixed it.

File: src/openrct2/localisation/StringIds.h [hunk 1]
```diff
@@ -3873,13 +3873,16 @@ enum
...
+    
 
     STR_CREATE_DUCKS = 6331,
     STR_REMOVE_DUCKS = 6332,
     STR_SHORTCUT_SCALE_UP = 6333,
     STR_SHORTCUT_SCALE_DOWN = 6334,
+    STR_OPTIONS_RANDOM_TITLE_SEQUENCE = 6335,
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    STR_COUNT = 32769
```
Comment (duncanspumpkin) [inline src/openrct2/localisation/StringIds.h]: This is how much space there is in the whole array. Don't change this value.

File: src/openrct2/localisation/StringIds.h [hunk 2]
```diff
@@ -3873,13 +3873,16 @@ enum
 
     STR_STRING_STRINGID = 6329,
     STR_DOWNLOADING_OBJECTS_FROM = 6330,
+    
+
```
Comment (duncanspumpkin) [inline src/openrct2/localisation/StringIds.h]: remove these extra blanks

**Comment by duncanspumpkin** [comment]
Note squash merge this to hide the merge commit.

**Comment by Gymnasiast** [comment]
@AaronVanGeffen If you add a changelog entry, you can merge it as far as I'm concerned.

**Comment by AaronVanGeffen** [comment]
Not entirely flawless, it seems:

**Pull request merged**